python/ray/exceptions.py

```diff
@@ -747,6 +747,13 @@ class ObjectRefStreamEndOfStreamError(RayError):
     pass
 
 
+@DeveloperAPI
+class ActorUnavailableError(RayError):
```

Comment:
Is there a way to distinguish whether (1) the called actor is unavailable because it cannot accept network traffic, or (2) if the calling actor is unavailable because it cannot send traffic? I doubt it is because from the caller's perspective these cases should look the same, but let me know otherwise.

Reply:
Unfortunately the cgroup based approach did not work. Now I changed to a break-the-connection approach so it does not matter of inbound or outbound - both breaks.